src/redux/actions/genes/loadDownsampledGeneExpression.js

```diff
@@ -55,7 +61,7 @@ const loadDownsampledGeneExpression = (
   const timeout = getTimeoutForWorkerTask(getState(), 'GeneExpression');
 
   try {
-    let requestETag;
+    let requestETag = null;
```

Comment:
looking at this if statement further down in this file:
```
    // If the ETag is different, that means that a new request was sent in between
    // So we don't need to handle this outdated result
    if (getState().genes.expression.downsampled.ETag !== requestETag) {
      return;
    }
```
based on it, shouldn't the default value of `requestETag` be equal to `getState().genes.expression.downsampled.ETag`? Because if the default value stays `null`, then we can have situations (if `onGeneratedETag` function is not called) where the if statement `if (getState().genes.expression.downsampled.ETag !== requestETag)` can return `true` simply because  the value of `requestETag` is not updated, not because a new request was sent in between.